kratos/python/add_containers_to_python.cpp

```diff
@@ -320,6 +320,8 @@ void  AddContainersToPython(pybind11::module& m)
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, RESIDUAL_NORM )
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, CONVERGENCE_RATIO )
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, BUILD_SCALE_FACTOR )
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, CONSTRAINT_SCALE_FACTOR )
```

Comment:
why don't we use the same as the build_scale_factor? is there a good reason for which we need fine control over those?

Reply:
The paper separates the tow types, de facto I consider the same value but may be different